### ISSUE TITLE
flatpak support on linux?

### ISSUE THREAD

**Issue by theoden8** [biolab/orange3#5403]
I imagine this will come up sooner or later, since electron apps are now being actively ported to it. flatpak has been gaining popularity as a way to sandbox desktop applications with their dependencies.

- [ ] **What's your use case?**

* My current set-up (debian sid) includes:
    * using a separate conda environment
    * bash script to activate that environment and start orange-canvas
    * desktop file stolen from one of the issues here
* One of the errors that inevitably came up was qt webkit/webengine also, which had to be resolved on the system level and on the conda environment level.

- [ ] **What's your proposed solution?** Linux flatpak

flatpak. why/why not:

* https://flatpak.org/
* https://flatkill.org/

Regarding pip/conda environments, this other pyqt project prefers this installation method:

* https://github.com/chipmuenk/pyfda/blob/develop/INSTALLATION.md

To summarise, it should be similar to osx app.

- [ ] **Are there any alternative solutions?**

* discussed in #1426

**Comment by janezd** [comment]
We discussed this at today's meeting. This would be nice to have, but it's not a priority and none of the core team expressed a particular desire to implement it. We're looking for volunteers. Would you like to do it?

The right repository for this would be https://github.com/ales-erjavec/orange3-installers.

**Comment by theoden8** [comment]
fine, I will give it a go after exam period, and see how much effort it involves

**Comment by theoden8** [comment]
Hey @janezd, a bit out of fashion this issue but i think i have something that runs, even if not perfect

https://github.com/theoden8/orange3-flatpak

it uses uv and https://github.com/flathub/com.riverbankcomputing.PyQt.BaseApp. Basically it installs uv from source and when you run the app it it installs orange3 from .lock file.  All extensions/updates you install go to the same venv and persist over multiple runs.

**Comment by theoden8** [comment]
@lanzagar I've posted some progress on discord and the PR to flathub is technically ready, just needs couple details and affiliation
